### PR TITLE
Consuming fix for property grid labels

### PR DESCRIPTION
**Pull request by cortneysmithbentley**
Consuming fix for property grid labels by updating @bentley/ui-components package and other related packages to 2.9.2 (fix is in 2.6, this is the latest non-dev release. was unsure which version to use)

### CODE REVIEW

File: packages/property-grid/package.json [hunk 1]
```diff
@@ -54,7 +54,7 @@
     "@bentley/presentation-frontend": "2.3.3",
     "@bentley/product-settings-client": "2.3.3",
     "@bentley/ui-abstract": "2.3.3",
-    "@bentley/ui-components": "2.3.3",
+    "@bentley/ui-components": "2.9.2",
```
Comment (aruniverse) [inline packages/property-grid/package.json:57]: you cant bump just one core pkg, its all or nothing. so please bump all the dev deps to 2.9.2, and the peer deps to ^2.9.2

**Review by aruniverse — CHANGES_REQUESTED** [review]
Maybe make this a minor bump rather than a patch

File: packages/property-grid/package.json (hunk 1 above)
Comment (cortneysmithbentley) [reply packages/property-grid/package.json:57]: gotcha didn't realize I missed some, will do.

Comment (cortneysmithbentley) [reply packages/property-grid/package.json:57]: Ok I definitely did this for the rest of the packages (just forgot the peer deps) and it didn't get committed, so I'm not totally useless. 👍  it's all pushed now

File: packages/property-grid/src/components/PropertyGrid.tsx [hunk 2]
```diff
@@ -28,6 +28,7 @@ import {
   PropertyGrid as CorePropertyGrid,
```
Comment (aruniverse) [inline packages/property-grid/src/components/PropertyGrid.tsx:28]: Is this import even being used anymore? If not, you can remove it. PropertyGrid as CorePropertyGrid

**Review by aruniverse — APPROVED** [review]

**Comment by aruniverse** [comment]
you also need to run `rush update` to update the lockfile.

**Review by aruniverse — APPROVED** [review]

**Pull request merged**